skbio/stats/distance/_base.py

```diff
@@ -43,71 +43,82 @@
 
     def __init__(self, missing_id):
         super(MissingIDError, self).__init__()
-        self.args = ("The ID '%s' is not in the dissimilarity matrix." % missing_id,)
+        self.args = ("The ID '%s' is not in the matrix." % missing_id,)
+
+
+class BaseMatrix(SkbioObject, PlottableMixin):
+    pass
 
 
-class DissimilarityMatrix(SkbioObject, PlottableMixin):
-    """Store dissimilarities between objects.
+class PairwiseMatrix(SkbioObject, PlottableMixin):
+    r"""Store pairwise relationships between objects.
 
-    A `DissimilarityMatrix` instance stores a square, hollow, two-dimensional
-    matrix of dissimilarities between objects. Objects could be, for example,
+    A `PairwiseMatrix` instance stores a square, two-dimensional
+    matrix of relationships between objects. Objects could be, for example,
     samples or DNA sequences. A sequence of IDs accompanies the
-    dissimilarities.
+    relationships.
 
-    Methods are provided to load and save dissimilarity matrices from/to disk,
-    as well as perform common operations such as extracting dissimilarities
+    Methods are provided to load and save pairwise matrices from/to disk,
+    as well as perform common operations such as extracting values
     based on object ID. Additionally, the
     :meth:`plot` method provides
     convenient built-in plotting functionality.
 
     Parameters
     ----------
-    data : array_like or DissimilarityMatrix
-        Square, hollow, two-dimensional ``numpy.ndarray`` of dissimilarities
-        (floats), or a structure that can be converted to a ``numpy.ndarray``
-        using ``numpy.asarray`` or a one-dimensional vector of dissimilarities
+    data : array_like or PairwiseMatrix
+        Square, two-dimensional `numpy.ndarray` of pairwise relationships
+        (floats), or a structure that can be converted to a `numpy.ndarray`
+        using `numpy.asarray` or a one-dimensional vector of pairwise relationships
         (floats), as defined by `scipy.spatial.distance.squareform`. Can
-        instead be a `DissimilarityMatrix` (or subclass) instance,
+        instead be a `PairwiseMatrix` (or subclass) instance,
         in which case the instance's data will be used.
-        Data will be converted to a float ``dtype`` if necessary. A copy will
-        *not* be made if already a ``numpy.ndarray`` with a float ``dtype``.
+        Data will be converted to a float `dtype` if necessary. A copy will
+        *not* be made if already a `numpy.ndarray` with a float `dtype`.
     ids : sequence of str, optional
         Sequence of strings to be used as object IDs. Must match the number of
-        rows/cols in `data`. If ``None`` (the default), IDs will be
+        rows/cols in `data`. If `None` (the default), IDs will be
         monotonically-increasing integers cast as strings, with numbering
-        starting from zero, e.g., ``('0', '1', '2', '3', ...)``.
+        starting from zero, e.g., `('0', '1', '2', '3', ...)`.
```

Comment:
Why this and below were changed from double ticks into single ticks?

When a docstring is prepended by "r", it will be rendered as [reStructuredText](https://docutils.sourceforge.io/rst.html) (RST), in which code blocks should be wrapped by double ticks, whereas single ticks represent italic instead.

Also, I think values like None, True, False don't need to be in a code block.